go/flags/endtoend/vtorc.txt

```diff
@@ -15,6 +15,7 @@ Usage of vtorc:
       --config-persistence-min-interval duration                    minimum interval between persisting dynamic config changes back to disk (if no change has occurred, nothing is done). (default 1s)
       --config-type string                                          Config file type (omit to infer config type from file extension).
       --consul_auth_static_file string                              JSON File to read the topos/tokens from.
+      --convert-tablets-with-errant-gtids                           Whether VTOrc should be changing the type of the tablets with errant GTIDs to DRAINED
```

Comment:
I wonder what the best descriptive name for this flag is.
1. disable-tablets-with-errant-gtids
2. drain-tablets-with-errant-gtids - The problem with this is that there is no real "draining" happening here

Reply:
At the risk of running too long, how about `--change-tablets-with-errant-gtid-to-drained`? It is the most accurate description of the actions taken.

Reply:
I think @shlomi-noach's suggestion is the best we have had so far. I have updated the PR with this name.